docs/source/datasets.rst

```diff
@@ -22,94 +22,102 @@ click the link provided in the "Module" column.
+| mridangam_stroke_  | Mridangam Stroke    | - audio: ✅         | - :ref:`stroke-name`      | 6977   |
+|                    |                     | - annotations: ✅   | - :ref:`tonic`            |        |
++--------------------+---------------------+---------------------+---------------------------+--------+
+| orchset_           | Orchset             | - audio: ✅         | - :ref:`melody` :ref:`f0` | 64     |
+|                    |                     | - annotations: ✅   |                           |        |
++--------------------+---------------------+---------------------+---------------------------+--------+
+| rwc_classical_     | RWC Classical       | - audio: ❌         | - :ref:`beats`            | 50     |
+|                    |                     | - annotations: ✅   | - :ref:`sections`         |        |
++--------------------+---------------------+---------------------+---------------------------+--------+
+| rwc_jazz_          | RWC Jazz            | - audio: ❌         | - :ref:`beats`            | 50     |
+|                    |                     | - annotations: ✅   | - :ref:`sections`         |        |
++--------------------+---------------------+---------------------+---------------------------+--------+
+| rwc_popular_       | RWC Pop             | - audio: ❌         | - :ref:`beats`            | 100    |
+|                    |                     | - annotations: ✅   | - :ref:`sections`         |        |
+|                    |                     |                     | - :ref:`vocal-activity`   |        |
+|                    |                     |                     | - :ref:`chords`           |        |
+|                    |                     |                     | - :ref:`tempo`            |        |
++--------------------+---------------------+---------------------+---------------------------+--------+
+| salami_            | Salami              | - audio: ❌         | - :ref:`sections`         | 1359   |
+|                    |                     | - annotations: ✅   |                           |        |
++--------------------+---------------------+---------------------+---------------------------+--------+
+| saraga_carnatic_   | Saraga Carnatic     | - audio: ✅         | - :ref:`tempo`            | 249    |
+|                    |                     | - multichannel: ✅  | - :ref:`phrases`          |        |
+|                    |                     | - annotations: ✅   | - :ref:`phrases`          |        |
+|                    |                     |                     | - :ref:`f0`               |        |
+|                    |                     |                     | - :ref:`f0` (vocal)       |        |
+|                    |                     |                     | - :ref:`sections`         |        |
+|                    |                     |                     | - :ref:`sections` (samas) |        |
```

Comment:
Yes, exactly. I stored the sama annotations as `SectionData` but it could also be stored as `EventData`. For who doesn't know exactly what _sama_ stands for: it's the timestamp of the beginning of a new rhythmic cycle (rhythmic cycles are called _talas_) or the end of the previous cycle. I thought that it would be more understandable for users to store _sama_ timestamps as `SectionData`, but I must say that I was doubtful about storing it as `EventData` as well. What are your thoughts about this @nkundiushuti? 
Sections here are the structural sections of the piece, typically there are between 2 and 4 sections approximately.

Reply:
it's fine to have it as SectionData. maybe we should put  :ref:`sections` (piece) :ref:`sections` (rhythmic,samas)

Reply:
Ok! Yes, this is going to be really helpful!